workflow/README.md

```diff
@@ -204,22 +271,22 @@ FIXME
 
 ``` ipldsch
 type SystemConfig struct {
-  secret Boolean (implicit False)
+  secret Boolean      (implicit False)
   check  Verification (implicit Attestation)
   time   Integer 
   memory Integer
   disk   Integer
 }
 
-type Verification enum {
+type Verification union {
   | Oracle
-  | Consensus(Integer)
-  | Optimistic(Integer)
+  | Consensus
+  | Optimistic
   | ZKP
-}
+} representation keyed
```

Comment:
Are user-defined verifications (e.g. probabilistic) contained in this concept? Maybe the "attestations" are designed to scale to that?